SPCTR-3271: Fix - Blocks goes underneath container background overlay pseudo element. (#3435)
brainstormforce/ultimate-addons-for-gutenberg: README.md

### x.x.x ###
* Fix: Modal - Resolved an issue in which the Modal would not open in Astra's quick-view popup for WooCommerce products.
* Fix: Separator - Resolved an edge-case issue in which the responsive alignment would not work as intended.

> * Fix: Advance Heading - Resolved an issue where heading block inside container goes underneath the ::before pseudo-element.

Reviewer: Let's move this changelog entry to the top, since it's for our Second Core Block.

```suggestion
* Fix: Heading - Resolved an issue wherein the Heading block inside Containers would appear underneath the Container's overlay when set.
```

Author: Done.

Reviewer: > Let's move this changelog entry to the top, since it's for our Second Core Block.

> Let's move this changelog entry to the top, since it's for our Second Core Block.

Can we move the Container entry on top as well? Since it's out first Core Block.

Author: Done.
